model/app.model.js

```diff
@@ -7,15 +7,22 @@ const selectTopics = () => {
     });
 };
 
-const selectArticle = (id) => {
+const selectArticleById = (id) => {
     const query = "SELECT * FROM articles WHERE article_id = $1;";
     let params = [id];
     return db.query(query, params).then((result) => {
         return result.rows;
     });
 };
 
+const selectArticles = () => {
+    return db.query("SELECT * FROM articles ORDER BY created_at DESC;").then((result) => {
```

Comment:
Here we want to Select only certain things (not body for instance) We also want to add a comment count column and test for it (which is the total count of all the comments with this article_id - you should make use of queries to the database in order to achieve this.)